packages/theme/colors.ts

```diff
@@ -1,133 +1,167 @@
 export type ThemeType = 'light' | 'dark';
 
 export const colors = {
-  apple: '#151E22',
-  google: '#E04238',
-  facebook: '#345997',
-  lightPrimary: '#000000',
-  darkPrimary: '#FFFFFF',
-  secondary: '#00D9D5',
-  danger: '#FF002E',
-  warning: '#FFEB14',
-  info: '#5398FF',
-  success: '#33FF2F',
-  light: '#EDEDED',
+  primaryLight: '#47498A',
+  secondaryLight: '#9E45B3',
+  acceptLight: '#6BF3C5',
+  infoLight: '#307EF1',
+  linkLight: '#A351F4',
+  successLight: '#0FC70C',
+  warningLight: '#EBBD1A',
+  dangerLight: '#F84444',
+
+  primaryDark: '#585DFA',
+  secondaryDark: '#C93AEB',
+  acceptDark: '#6BF3C5',
+  infoDark: '#2998FF',
+  linkDark: '#A351F4',
+  successDark: '#2FFA86',
+  warningDark: '#F4CC3E',
+  dangerDark: '#FF3C3C',
+
+  white: '#FFFFFF',
+  gray1: '#EDEDED',
+  gray2: '#CCCCCC',
+  gray3: '#BEBEBE',
+  gray4: '#909090',
+  gray5: '#787878',
+  gray6: '#4D4D4D',
+  gray7: '#474747',
+  gray8: '#282828',
+  gray9: '#1E1E1E',
+  black: '#000000',
+  red1: '#FF2C2C',
+  red2: '#DA0000',
 };
 
 export const light = {
   bg: {
-    default: '#FFFFFF',
-    paper: '#EDEDED',
-    disabled: '#C4C4C4',
-    border: 'rgba(0,0,0,0.2)',
+    basic: colors.white,
+    paper: colors.gray1,
+    disabled: colors.gray3,
+    border: colors.gray2,
+    borderContrast: colors.gray7,
   },
   role: {
-    primary: colors.lightPrimary,
-    secondary: colors.secondary,
-    danger: colors.danger,
-    warning: colors.warning,
-    info: colors.info,
-    success: colors.success,
-    light: colors.light,
+    primary: colors.primaryLight,
+    secondary: colors.secondaryLight,
+    accept: colors.acceptLight,
+    info: colors.infoLight,
+    link: colors.linkLight,
+    success: colors.successLight,
+    warning: colors.warningLight,
+    danger: colors.dangerLight,
   },
   text: {
-    default: '#000000',
-    primary: '#000000',
-    placeholder: '#6D6D6D',
-    disabled: '#C4C4C4',
-    contrast: '#FFFFFF',
-    validation: '#FF002E',
-    secondary: '#404040',
-    link: '#393D7A',
-    accent: '#B446BF',
+    basic: colors.black,
+    label: colors.gray6,
+    placeholder: colors.gray5,
+    disabled: colors.gray3,
+    validation: colors.red2,
+    contrastBasic: colors.white,
+    contrastLabel: colors.gray3,
+    contrastPlaceholder: colors.gray4,
+    contrastDisabled: colors.gray6,
+    contrastValidation: colors.red1,
```

Comment:
Let's remove everything except `contrastBasic`, and this can be renamed to `contrast`.